### PR TITLE
Animations

### PR DESCRIPTION
**Pull request by segerphilip**
Implements a ripple circle animation that expands outwards. Also integrates the animation with the login page screen to show sort of the mechanics of the game.

### CODE REVIEW

File: app/src/main/java/website/bloop/app/Circle.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+        // Set color for drawing the triangle
+        GLES20.glUniform4fv(mColorHandle, 1, color, 0);
+
+        mMVPMatrixHandle = GLES20.glGetUniformLocation(mProgram, "uMVPMatrix");
+
+        // Apply the projection and view transformation
+        GLES20.glUniformMatrix4fv(mMVPMatrixHandle, 1, false, mvpMatrix, 0);
+
+//        GLES20.glDrawArrays(GLES20.GL_TRIANGLES, 0, 364);
```
Comment (wolfd) [inline app/src/main/java/website/bloop/app/Circle.java]: forgot about this?

File: app/src/main/java/website/bloop/app/PlayLoginActivity.java [hunk 2]
```diff
@@ -70,16 +68,23 @@ public void onConnected(@Nullable Bundle bundle) {
...
     @Override
     public void onConnectionSuspended(int i) {
+        Log.d(TAG, "onConnectionSuspended()");
         // Attempt to reconnect
         mGoogleApiClient.connect();
     }
 
     @Override
     public void onConnectionFailed(@NonNull ConnectionResult connectionResult) {
+        Log.d(TAG, "onConnectionFailed()");
```
Comment (wolfd) [inline app/src/main/java/website/bloop/app/PlayLoginActivity.java]: Do we still want these?

**Review by wolfd — CHANGES_REQUESTED** [review]

File: app/src/main/java/website/bloop/app/Circle.java (hunk 1 above)
Comment (segerphilip) [reply app/src/main/java/website/bloop/app/Circle.java]: yep :-1:

File: app/src/main/java/website/bloop/app/PlayLoginActivity.java (hunk 2 above)
Comment (segerphilip) [reply app/src/main/java/website/bloop/app/PlayLoginActivity.java]: nope :-1:

**Review by wolfd — APPROVED** [review]
Looks dank

**Pull request merged**